styles.css

```diff
@@ -77,3 +77,168 @@ nav {
   align-items: center;
   color: #505f79;
 }
+
+.work {
+  margin: 25px 0;
+  display: grid;
+  row-gap: 25px;
+  padding: 100px 10px 50px 10px;
+  background: #e5e5e5;
```

Comment:
:red_circle:  [OPTIONAL] Please make sure you pick the right background value from Figma Design for your work section
This should be the right hex code [`#F7F7F9 `] value